src/content/docs/kubernetes-pixie/k8s-otel/advance-config.mdx

```diff
@@ -0,0 +1,266 @@
+  ```
+
+
+## Send data via a proxy [#proxy-config]
+
+To send your Kubernetes telemetry data through a proxy, you can configure the OpenTelemetry Collector to use an HTTP proxy for outbound connections. This is particularly useful in environments where direct internet access is restricted or monitored.
+
+1. (Option 1): Use the `proxy` section in your `values.yaml` file to the `nr-k8s-otel-collector` chart configuration.
+
+  ```yaml
+    proxy: '<Your-proxy-server-URL>' # Example: [http://squid-proxy.squid:3128](http://squid-proxy.squid:3128)
+  ```
+
+2. (Option 2): The OpenTelemetry Collector also respects standard proxy environment variables (`HTTP_PROXY`, `HTTPS_PROXY`, `NO_PROXY`). You can inject these directly into the collector pods, optionally pulling values from Kubernetes Secrets for enhanced security. Note that these settings must apply to both the `deployment` and `daemonset` collectors. For more information on OpenTelemetry proxy settings, refer to the [OpenTelemetry Collector documentation](https://opentelemetry.io/docs/collector/configuration/).
+
+    * To inject proxy settings via environment variables in the `env` section of your `values.yaml` file:
+
+      ```yaml
+      daemonset:
+        envs:
+          - name: HTTPS_PROXY
+            valueFrom:
+              secretKeyRef:
+                name: <Your Secret Name> # Name of your Kubernetes Secret
+                key: <Your Secret Key>    # Key within the Secret that holds the proxy URL
+      deployment:
+        envs:
+          - name: HTTPS_PROXY
+            valueFrom:
+              secretKeyRef:
+                name: <Your Secret Name>
+                key: <Your Secret Key>
+      ```
+
+    * To inject proxy settings via environment variables in the `envFrom` section of your `values.yaml` file:
+
+      ```yaml
+      daemonset:
+        envsFrom:
+          - secretRef:
+              name: <Your Secret Name>
+      deployment:
+        envsFrom:
+          - secretRef:
+              name: <Your Secret Name>
+      ```
+  
+
+
+## Add custom configurations in the Helm chart [#custom-config]
+
+The `extraConfig` sections within the `values.yaml` file provide a powerful way to extend the functionality of both the `daemonset` and `deployment` collectors. Depending on your specific requirements, you can choose either collector to apply additional configurations, allowing you to tailor your monitoring experience.
+
+These options offer flexibility for integrating specific settings not included by default. 
+
+{/* For a comprehensive list of receivers, processors, exporters, and pipelines that you can reuse in your configurations, refer to [](#how-works). */}
+
+For more details, always review the `values.yaml` file in the [Helm chart repository](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/values.yaml).
+
+
+## Enable Prometheus service discovery [#enable-prom-sd]
```

Comment:
We may consider to state that this section is just an example of adding your custom configuration to the Helm Chart